src/attacks/tests/test_attackconsole.py

```diff
@@ -75,6 +78,12 @@ def test_run(self, ac: SubAttackConsole):
         ac.onecmd("run")
         assert ac.attack.run.called
 
+    @patch("attacks.Attack.handle_interrupt")
+    def test_keyboard_interrupt_is_caught(self, ac: SubAttackConsole):
+        FakeAttack.run = Mock(side_effect=lambda: _raise(KeyboardInterrupt))
+        ac.onecmd("run")
+        assert True  # No Exception is thrown
```

Comment:
Isn't this line superfluous?

Reply:
tbh, I just copied this from the other exception test (directly below this one). We could also combine this into one test if you want

Reply:
I guess it's OK, I just found it strange to assert True, but maybe that's some convention.